WordPress/Classes/ViewRelated/Post/PostAuthorSelectorViewController.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+
+@objc class PostAuthorSelectorViewController: SettingsSelectionViewController {
+    /// The post to change the author.
+    private var post: AbstractPost!
```

Comment:
```suggestion
    private let post: AbstractPost
```
If this is always populated by the initializer, I think it can be a constant. Wdyt?

Reply:
I touched on this a little here regarding the override inits: https://github.com/wordpress-mobile/WordPress-iOS/pull/16281#discussion_r619296509

Reply:
Thanks again, looks like these are needed. Resolving this thread.